posthog/temporal/data_imports/pipelines/pipeline.py

```diff
@@ -26,16 +27,39 @@ class PipelineInputs:
 class DataImportPipeline:
     loader_file_format: Literal["parquet"] = "parquet"
 
-    def __init__(self, inputs: PipelineInputs, source: DltSource, logger: FilteringBoundLogger):
+    def __init__(
+        self, inputs: PipelineInputs, source: DltSource, logger: FilteringBoundLogger, incremental: bool = False
+    ):
         self.inputs = inputs
         self.logger = logger
-        self.source = source
+        # Assuming each page is 100 items for now so bound each run at 50_000 items
+        self.source = source.add_limit(500)
```

Comment:
I don't fully understand this comment - this will limit the source to 500 items for all sources, right?